📝 Update docs about hypothesis (#6266)
pydantic/pydantic: docs/integrations/hypothesis.md

Pydantic v2.0 drops built-in support for Hypothesis and no more ships with the integrated Hypothesis plugin.

!!! warning

>     We are temporarily removing the Hypothesis plugin in favor of studying a different mechanism. See more about it on [annotated-types/annotated-types#37](https://github.com/annotated-types/annotated-types/issues/37).

Reviewer: ```suggestion
    We are temporarily removing the Hypothesis plugin in favor of studying a different mechanism. For more information, see the issue [annotated-types/annotated-types#37](https://github.com/annotated-types/annotated-types/issues/37).
```
